contrib/video_server/video_server.py

```diff
@@ -148,7 +158,7 @@ def main():
         help='Specify a video file path, rtsp stream address or camera value')
     parser.add_argument('-p', '--port', default=6420, type=int)
     parser.add_argument('-a', '--address', default="0.0.0.0")
-
+    parser.add_argument('-s', '--shape', default=None, nargs='+', type=int)
```

Comment:
might need some help text to make tuple input more apparent

Reply:
I added a help me.